src/main/java/org/jahia/modules/sam/healthcheck/probes/SupportedStackTomcatProbe.java

```diff
@@ -0,0 +1,55 @@
+package org.jahia.modules.sam.healthcheck.probes;
+
+import org.jahia.commons.Version;
+import org.jahia.modules.sam.Probe;
+import org.jahia.modules.sam.ProbeSeverity;
+import org.jahia.modules.sam.ProbeStatus;
+import org.jahia.settings.SettingsBean;
+import org.osgi.service.component.annotations.Component;
+
+import java.util.Map;
+
+@Component(immediate = true, service = Probe.class)
+public class SupportedStackTomcatProbe implements Probe {
+
+    Version supportedTomcatVersion = new Version("9.0.89");
```

Comment:
Right now I see it set as 9.0.85 https://github.com/Jahia/jahia-private/blob/master/jahia-parent/pom.xml#L113, so maybe we need to set the value.

Also I'm wondering if we should maybe expose/set this value in the pom properties to make it consistent with how we declare tomcat versions and make maintenance easier (maybe up for discussion). 

One way would be to set up a system variable in maven and use `System.getProperty(key)` in java to retrieve it (https://stackoverflow.com/a/11500648)